bezier/src/cubic_bezier.rs

```diff
@@ -243,26 +310,19 @@ impl CubicBezierSegment {
         return max_t;
     }
 
-    /// Find the advancement of the x-least position in the curve.
-    ///
-    /// This returns the advancement along the curve, not the actual x position.
+    /// Find the x-least position in the curve.
     pub fn find_x_minimum(&self) -> f32 {
         let mut min_t = 0.0;
         let mut min_x = self.from.x;
         if self.to.x < min_x {
             min_t = 1.0;
             min_x = self.to.x;
         }
-        for t in self.find_inflection_points() {
```

Comment:
You should keep this loop, but replace `find_inflection_points` with `find_x_inflections`. Otherwise, if the control point is left of the start and end points (in which case the curve will go more to the left of from and to), this would not return the correct result.
`find_x_maximum` and `find_y_minimum/maximum` also need to be fixed up to use `find_x/y_inflections` instead of find_inflection_points`.